variables.tf

```diff
@@ -0,0 +1,80 @@
+/*
+Module variables
+*/
+
+variable "alb_is_internal" {
+  description = "Determines if the ALB is internal. Default: false"
```

Comment:
Usually there is o need to write what is the default value, because it is visible 1 line below.

Reply:
Removed where relevant.